clusterloader2/README.md

```diff
@@ -1,21 +1,62 @@
 # ClusterLoader
 
-### Running ClusterLoader
+## Running ClusterLoader
 
 To run ClusterLoader type:
 ```
-go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.json
+go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.yaml --nodes=<number_of_nodes>
```

Comment:
Why do we need "nodes"?

Can't we infer it from what "kubectl get nodes" returns?